src/patterns/ec2-app/base.ts

```diff
@@ -128,25 +128,6 @@ export interface GuEc2AppProps extends AppIdentity {
   certificateProps: StageAwareValue<GuDomainName>;
 }
 
-const OpenApplicationCidrError: Error =
-  Error(`Your list of CIDR ranges includes 0.0.0.0/0, meaning all other ranges specified are unnecessary as
-your application is open to the world. Please either remove the open CIDR range or use the PUBLIC access scope.`);
-
-const PublicCidrError: Error =
-  Error(`Your list of CIDR ranges includes a public ip, which will not be able to reach an internal load balancer.
-  Please either remove the public CIDR range or use the RESTRICTED access scope.`);
-
-function validateRestrictedCidrRanges(access: RestrictedAccess) {
-  const openToWorld = access.cidrRanges.map((range) => range.uniqueId).includes("0.0.0.0/0");
-  if (openToWorld) throw OpenApplicationCidrError;
-}
-
-function validateInternalCidrRanges(access: InternalAccess) {
-  const publicCidrRanges = access.cidrRanges.filter((range) => !range.uniqueId.startsWith("10."));
-  const containsPublicIps = publicCidrRanges.length > 0;
```

Comment:
re-written as [`some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) for brevity